src/main/java/cz/inventi/kpj/spring/springintroduction/controllers/AppController.java

```diff
@@ -0,0 +1,20 @@
+package cz.inventi.kpj.spring.springintroduction.controllers;
+
+import cz.inventi.kpj.spring.springintroduction.service.GreetingService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
The name of the service is confusing... You will learn about the RestControler in the next lesson. Now you should use the service but it will work.

I think that you know the framework and this example was too simple for you.